application/values.yaml

```diff
@@ -14,31 +14,31 @@ applicationName: "application"
 
 ##########################################################
 # Global labels
-# These labels will be added on all resources, 
-# and you can add additional labels from below 
+# These labels will be added on all resources,
+# and you can add additional labels from below
 # on individual resource
 ##########################################################
-  
-cronJob: 
+
+cronJob:
```

Comment:
Add `priorityClass` to cronjob too (in the commented `db-migration` block)

Reply:
@kseniyashaydurova 
Done